### PR TITLE
fix name of nb and title

### PR DESCRIPTION
**Pull request by jperez999**

### CODE REVIEW

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

File: examples/Getting_Started/Serving-Ranking-Models-With-Merlin-Systems.ipynb [hunk 1]
```diff
@@ -30,7 +30,7 @@
    "source": [
```
Comment (rnyak) [inline examples/Getting_Started/Serving-Ranking-Models-With-Merlin-Systems.ipynb:30]: <p>May be remove this sentence <code>*<em>Be sure to check the other components of the Merlin framework, they can help you *</em></code></p><p> </p><p> </p><p><code><em>we do not have </em></code>&nbsp;<a href="https://github.com/NVIDIA-Merlin/NVTabular" target="_blank">NVTabular</a> example for Ali-CCP dataset.  may be rephrased that sentence as " to see how the data is transformed by NVTAbular, check out this notebook https://github.com/NVIDIA-Merlin/models/blob/main/examples/04-Exporting-ranking-models.ipynb"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

**Comment by rnyak** [comment]
@jperez999 can you pls add the readme.md that I sent you to this PR? thanks.

**Comment by jperez999** [comment]
rerun tests

**Review by mikemckiernan — APPROVED** [review]
Thanks for revising the title.

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Pull request merged**